lib/iceserversource/nts.js

```diff
@@ -92,6 +92,10 @@ class NTSIceServerSource extends EventEmitter {
       _log: {
         value: log
       },
+      _status: {
```

Comment:
@markandrus Any danger of setting the default value to "failure" here?

Reply:
Hmm... the calling code in `connect` is supposed to ensure we get a first ICE server emitted from the IceServerSource before proceeding. So long as we retain that contract, there should be no danger here. Alternatively, we could allow for `null`. WDYT?

Reply:
@markandrus I think it would be nice to have a separate default value. That way, if we report it in the "connect" message, we know that it's actually not a failure, but some other problem. Ideally we should never report the default value.

Reply:
OK, let's allow for `null` now.